Simplenote/Classes/OptionsViewController.swift

```diff
@@ -386,6 +387,7 @@ private extension OptionsViewController {
         SPTracker.trackEditorCopiedPublicLink()
         UIPasteboard.general.copyPublicLink(to: note)
         NoticeController.shared.present(NoticeFactory.linkCopied())
+        SPTracker.trackNoticeLinkCopied()
```

Comment:
`copyInterlinkWasPressed` also calls `SPTracker.trackNoticeLinkCopied`

Note that:

- Copy Interlink: Copies the Simplenote internal reference to a note
- Copy Link (this precise API) copies the **Public Link**. That is the published URL

Perhaps this one should be `trackPublicLinkCopied` and `trackNoticeLinkCopied` should be `trackInterlinkCopied` ?

Reply:
Hmmm, I think I am not clear on what are we trying to track.  

If we are tracking that the notice has been displayed, both notices for public link and inter link are the same.  So I would think that we wouldn't want to differentiate the two.  And since public link copied and interlink copied also create track events, it feels a bit like duplication to me.

But I can also see how seeing the extra details of which action caused the creation of the copy link notice to appear could be useful.  🤔 

@SylvesterWilmott do you have any thoughts on the different types of "Link Copied" notices.  Should they all be treated the same?  Or should the different types of Link Copied be noted with what link was copied

Reply:
Sorry for the delay on this. Tracking only the type of notice is okay but we should differentiate between public link and internal link. We can use one event for both since the notices are identical and a "type" property to distinguish between the two.